CMSgov/easi-app
# [EASI-3447] Ensure intake.SubmittedAt is set when re-submitting an intake

# EASI-3447

## Changes and Description

- Ensures that `intake.SubmittedAt` is set every time the Submit Action is taken (not just if we're submitting for the first time)

## How to test this change

- Submit an intake
- Note the `submitted_at` field in the DB
- Request edits on the intake
- Manually take an action in GQL to re-submit the intake with `submitIntake`
- Ensure that the `submitted_at` field in the DB was properly updated.

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
